src/component/Animatable.lua

```diff
@@ -0,0 +1,16 @@
+--Animatable.lua
```

Comment:
My personal preference is to assign either a sprite or animation to the renderable and then on draw check with getAssetType which actions to perform. As both animations and images are both "renderable"